android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java

```diff
@@ -0,0 +1,165 @@
+package com.liferay.mobile.screens.viewsets.defaultviews.ddl.form.fields;
+
+import android.content.Context;
+import android.os.Build;
+import android.support.annotation.Nullable;
+import android.support.annotation.RequiresApi;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import com.liferay.mobile.screens.R;
+import com.liferay.mobile.screens.ddl.form.view.DDLFieldViewModel;
+import com.liferay.mobile.screens.ddl.model.GeoLocation;
+import com.liferay.mobile.screens.ddl.model.GeolocationField;
+import com.liferay.mobile.screens.util.LiferayLocale;
+import java.text.NumberFormat;
+import java.text.ParseException;
+
+/**
+ * @author Víctor Galán Grande
+ */
+
+public class DDLFieldGeoView extends LinearLayout implements DDLFieldViewModel<GeolocationField>, TextWatcher {
+
+
+	public static final double INVALID_LATITUDE = -91;
+	public static final double INVALID_LONGITUDE = -181;
+	protected GeolocationField field;
+	protected TextView labelTextView;
+	protected EditText latitudeEditText;
+	protected EditText longitudeEditText;
+	protected View parentView;
+
+	public DDLFieldGeoView(Context context) {
+		super(context);
+	}
+
+	public DDLFieldGeoView(Context context, @Nullable AttributeSet attrs) {
+		super(context, attrs);
+	}
+
+	public DDLFieldGeoView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
+		super(context, attrs, defStyleAttr);
+	}
+
+	@RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+	public DDLFieldGeoView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+		super(context, attrs, defStyleAttr, defStyleRes);
+	}
+
+	@Override
+	public GeolocationField getField() {
+		return field;
+	}
+
+	@Override
+	public void setField(GeolocationField field) {
+		this.field = field;
+
+		if (this.field.isShowLabel()) {
```

Comment:
if (labelTextView != null) {
  labelTextView.setVisibility(this.field.isShowLabel() ? VISIBLE : GONE);
labelTextView.setText(this.field.getLabel());
}
?

Reply:
I'll buy it :P